vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -388,6 +388,17 @@ public void addKeyStrokeListener(KeyStrokeListener l) {
       l.addKeyStrokeSource(s);
     }
   }
+  
+  
+  /**
+   * If our keyboard mapping paradigm changes (example: Mac Legacy preference checked/unchecked), we need to reregister all of our KeyStrokeListeners 
+   */
+  public void refreshKeyStrokeListeners() {
+    for (KeyStrokeListener kl : keyStrokeListeners) {
+      kl.setKeyStroke(kl.getKeyStroke());
+    }
```

Comment:
This could be done as `keyStrokeListeners.forEach(l -> l.setKeyStroke(l.getKeyStroke));`. Slightly shorter that way.

Reply:
Except for the typo at the end, but yeah ;-)   Checked in.